src/components/single-place/main-section/add-to-wish-list-bttn/add-to-wish-list-bttn.js

```diff
@@ -1,17 +1,80 @@
 import React, { Component } from 'react';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faPlus } from '@fortawesome/free-solid-svg-icons';
 import './add-to-wish-list-bttn.scss';
+import PlansListService from '../../../../services/plans-list-service';
 
 export default class AddToWishListBttn extends Component {
-   render() {
-      return(
-        <div className ='add-to-wish-list-bttn'>
-            <FontAwesomeIcon icon={faPlus}
-             size="2x"
-             title='Add place to my Wish List'
-             />
-         </div>
-      )
-   }
-}
\ No newline at end of file
+  service = new PlansListService();
+
+  state = {
+    myPlansList: [],
+    wishId: null,
+    wishState: false,
+  }
+
+  componentDidMount() {
+    this.service.getPlansList()
+      .then((plansList) => {
+        const find = plansList.find(place => {
```

Comment:
bad naming of variables. It is unclear what "find" means without context.
findPlace or smth like this better